dist/c2fo-ui-components.css

```diff
@@ -0,0 +1,45 @@
+.pager {
```

Comment:
As a note to others looking at this. 

I expect this CSS will eventually get modified and/or moved into the ui-guide repository as we get further along.